### ISSUE TITLE
Opening DB with lots of keys hangs

### ISSUE THREAD

**Issue by bp74** [redis/RedisDesktopManager#3665]
RDM Version: 0.8.7.317<br>
Environment (OS name and version): Windows 8.1<br>
Redis Server Version: 3.0.0<br>

Steps to reproduce:
1. Connect to the server
2. Open db0 which contains lots of keys (my db0 has 215165 keys)
3. Watch RDM to hang

Expected result: 
RDM should load all keys (even if it takes some time) and then show the tree

Actual Result: 
RDM starts loading the keys but stops after a few thousand keys.

Here is the output from the log window:

```
2016-08-02 10:23:52 : Connection: lx-devredis01
(DEV) > [runCommand] scan 0 MATCH * COUNT 10000
2016-08-02 10:23:53 : Connection: lx-devredis01
(DEV) > Response received : Array
2016-08-02 10:23:53 : Connection: lx-devredis01
(DEV) > [runCommand] scan 132144 MATCH * COUNT 10000
.... waiting forever .....
```

If you try again and you are lucky, it works a little bit longer:

```
2016-08-02 10:25:38 : Connection: lx-devredis01
(DEV) > [runCommand] scan 0 MATCH * COUNT 10000
2016-08-02 10:25:38 : Connection: lx-devredis01
(DEV) > Response received : Array
2016-08-02 10:25:38 : Connection: lx-devredis01
(DEV) > [runCommand] scan 1072 MATCH * COUNT 10000
2016-08-02 10:25:38 : Connection: lx-devredis01
(DEV) > Response received : Array
2016-08-02 10:25:38 : Connection: lx-devredis01
(DEV) > [runCommand] scan 202776 MATCH * COUNT 10000
2016-08-02 10:25:39 : Connection: lx-devredis01
(DEV) > Response received : Array
2016-08-02 10:25:39 : Connection: lx-devredis01
(DEV) > [runCommand] scan 141348 MATCH * COUNT 10000
.... waiting forever .....
```

Sometimes it also happens that RDM does not hang, but crashes in the same situation.

**I did the same test with RDM 0.8.6.270 and this version worsk fine!**

**Comment by uglide** [comment]
hello @bp74 
Do you use SSH tunnel?

**Comment by bp74** [comment]
No, just a plain connection.

**Comment by bp74** [comment]
It is a remote server, so there is a little bit of latency ... maybe this makes a difference?!

**Comment by uglide** [comment]
@bp74 Could you please provide output of command `config get timeout`

**Comment by bp74** [comment]
Here is the output, that's interessting:

lx-devredis01:6365> config get timeout
1) "timeout"
2) "0"

**Comment by uglide** [comment]
Ok, it's not timeout issue 
@bp74  Could you please try this night build https://ci.appveyor.com/project/uglide/redisdesktopmanager/build/0.8.7.344/artifacts ?

**Comment by bp74** [comment]
Wow, the nightly build works perfect!
I will use it a little bit and keep you up to date.

**Comment by uglide** [comment]
I released 0.8.7-1 with fixed qredisclient.
Feel free to re-open this issue or open new one if you experiencing any problems.

**Comment by bp74** [comment]
Thanks a lot!
